### ISSUE TITLE
GitCommitBear: Enforce only a single newline between shortlog and body

### ISSUE THREAD

**Issue by Makman2** [coala/coala-bears#1351]
This is bad:
```
shortlog




body
```

**Comment by gs0510** [comment]
@arnab1896  Are you still working on this?

**Comment by arnab1896** [comment]
Yes, I will be from 12th Feb, sorry.

**Comment by arnab1896** [comment]
* I mean, yes I'm working on it

**Comment by Monal5031** [comment]
@arnab1896 Are you still working on it? If not @Makman2 please assign me to it :).

**Comment by Makman2** [comment]
Reassigning due to inactivity.

**Comment by Monal5031** [comment]
@Makman2 He had an open PR though https://github.com/coala/coala-bears/pull/1450 should we close it too?

**Comment by jackthepirate6** [comment]
I would like to take this issue

**Comment by Monal5031** [comment]
@jackthepirate6 I've already sent a PR and made the required changes but the issue here is that when we do `self.git_commit(commit)` it somehow deletes the extra newlines such that there can be maximum of 2 consecutive, thats why the tests are failing.

**Comment by jackthepirate6** [comment]
ok

**Comment by Makman2** [comment]
I've actually tested that with git, and it seems git automatically removes redundant newlines. So passing a message like
```
Shortlog




Body
```
can never occur, because git postprocesses the message. But what's possible is
```
Shortlog
Body
```
Then it keeps the message as is without a blankline.

**Comment by Monal5031** [comment]
Yes that is infact what I earlier reported in the gitter chat sometime near start of GCI @Makman2
So 
```
HEAD

Shortlog
```
Is the expected commit message, thus making it specific to have two newlines, please correct me if I am going the wrong way.

**Comment by Makman2** [comment]
Yeah this issue is about to implement a setting enforcing this.
